### PR TITLE
[debian] Add symlink and tflchef tool

### PR DESCRIPTION
**Pull request by mhs4670go**
This commit adds symlinks and tflchef tool to one-compiler-dev package.

Signed-off-by: seongwoo <mhs4670go@naver.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
@nnfw-bot test cross-runtime-test-debug

File: debian/one-compiler-dev.install [hunk 1]
```diff
@@ -2,5 +2,6 @@
 # bin
 usr/bin/circledump usr/share/one/bin/
 usr/bin/circle-tensordump usr/share/one/bin/
+usr/bin/tflchef usr/share/one/tflchef
```
Comment (seanshpark) [inline debian/one-compiler-dev.install:5]: Um? I thought you didn't need this...

Comment (jyoungyun) [reply debian/one-compiler-dev.install:5]: @seanshpark The `tflchef` has released in NPU_Compiler. So I asked him to add this file.

@mhs4670go `usr/share/one/bin`?

Comment (seanshpark) [reply debian/one-compiler-dev.install:5]: maybe add to `one-compiler-dev.links` too?

Comment (seanshpark) [inline debian/one-compiler-dev.install:5]: > @mhs4670go usr/share/one/bin?

@mhs4670go, have you checked this?

Comment (mhs4670go) [reply debian/one-compiler-dev.install:5]: oh..

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by jyoungyun — APPROVED** [review]
LGTM

**Pull request merged**